_posts/2020-04-11-native-mobile-to-flutter.markdown

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: "Looking Back on Our Migration from Native Mobile to Flutter."
```

Comment:
The title is descriptive, which is good, but it's a bit long from an attention-grabbing perspective.   Maybe it's just the layout of our current blog but it also wraps in places which is not ideal.   I think we could drop "Looking Back on" to simplify, and compensate for it in the post's intro.